scripts/ci/deploy.sh

```diff
@@ -48,7 +48,7 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$ARGV_ARCHITECTURE" \
     -s "$(pwd)" \
-    -c "make publish-aws-s3"
+    -c "make publish-aws-s3 publish-bintray-debian publish-bintray-redhat"
```

Comment:
As I mentioned in an earlier comment, IMHO both the `if` and `else` branches of this script should simply be doing `make publish-all`, and then the `Makefile` should be modified to build up that target dynamically (I guess via e.g. a `PUBLISH_TARGETS` variable).